src/main/java/eu/proteus/producer/model/SensorMeasurement.java

```diff
@@ -12,7 +12,7 @@
 	protected double x;
 
 	public SensorMeasurement() {
-		this.type = this.getClass() == SensorMeasurement2D.class ? (byte) 0x0001f : (byte) 0x0000f;
+		this.type = this.getClass() == SensorMeasurement2D.class ? (byte) 0x0001 : (byte) 0x0000;
```

Comment:
There are not 'hex strings'. We have pure byte values (0x00 and 0x01) and we are casting them to byte, even though they are already bytes (probabily jvm is ignoring the explícitos castings, but it is not readable at all...